Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -4417,34 +4417,28 @@ private void HighlightElement(ElementInfo ElementInfo, bool locateElementByItLoc
                 SwitchFrame(ElementInfo.Path, ElementInfo.XPath, true);
 
                 //Find element 
-
-                if ((IWebElement)ElementInfo.ElementObject == null)
+                if (locateElementByItLocators)
```

Comment:
why IF was removed? that means we will keep calling to driver to get the IWebElement even though we already have it, no?